### ISSUE TITLE
Save as image

### ISSUE THREAD

**Issue by amjad** [jharwig/PPSSignatureView#26]
I use this in my app and wanted to know if there's a way to save the signature as an image (base64 encoded) to send via API to our payment gateway server. Thanks.

**Comment by amjad0** [comment]
@jharwig

**Comment by ghufi** [comment]
hi @jharwig  @amjad  did you fine any solution of this if yes please let me know 

Thanks

**Comment by amjad** [comment]
@ghufi Yes we did. We converted it to a base64 encoded image and saved it locally.

**Comment by ghufi** [comment]
Thank you for reply , i also did that by taking snapshot (**programmatically**) of the signature view and then save that image to server and it works
